### DISCUSSION TITLE
Why not whole JavaScript snippet is syntax hightlighted?

### DISCUSSION THREAD

**Discussion by jcubic** [facebook/docusaurus#10777] [Q&A] [answered]
Is this a bug? My JavaScript object literal is not syntax highlighted.

![Przechwycenie obrazu ekranu_2024-12-17_23-48-37](https://github.com/user-attachments/assets/e521704e-368b-45d7-b5ea-c6fec2bd54c1)

The code works fine with Prism.js: https://prismjs.com/test.html#language=javascript

This is my code:

```javascript
$('body').terminal({
    hello(what = 'world') {
        this.echo(`hello, this is ${what}!`);
    },
    bye() {
        this.echo('goodby');
    }
}, {
    checkArity: false
});
```

Even GitHub don't have issues highlighting this code.

**Comment by Josh-Cena** [comment, marked as answer]
If you look at the inspected DOM, you see that the tokens do get correct class names like `operator` and `boolean`. However, the particular theme you've chosen does not give any of these tokens special colors. Either choose another theme, or enhance this theme, following our guide here: https://docusaurus.io/docs/markdown-features/code-blocks#theming
